File: Interviews/GFG/Daily-Problems/2022/November/Day-11/Primes-sum.py
class Solution:
    def isPrime(self, N) -> bool:
        i = 2
        while i * i <= N:
            if N % i == 0:
                return False
            i += 1
        return True

    def isSumOfTwo(self, N):
        for i in range(2, N):
            if self.isPrime(i) and self.isPrime(N - i):
                return "Yes"

        return "No"

File: Interviews/GFG/Daily-Problems/2022/November/Day-11/Primes-sum.cpp
class Solution {
   public:
    bool isPrime(int N) {
        for (int i = 2; i * i <= N; i++) {
            if (N % i == 0)
                return false;
        }
        return true;
    }
    string isSumOfTwo(int N) {
        for (int i = 2; i <= N - 2; i++) {
            if (isPrime(i) and isPrime(N - i)) {
                return "Yes";
            }
        }

        return "No";
    }
};